src/index.js

```diff
@@ -163,8 +163,29 @@ const createRepoRunnerTokenFn = suffix => async opts => {
   log(`Creating repo runner ${suffix} token`);
   return (await client.post(`/repos/${repoName}/actions/runners/${suffix}-token`)).data.token
 };
+const createEntRunnerTokenFn = suffix =>  async opts => {
+  if (opts.repoName && !opts.entName)
+    opts.entName = opts.repoName.split('/')[0];
+
+  const { entName, parentLog = rootLog } = opts;
+  if (!entName) // TODO we could get this from installation
+    throw new ArgumentError('Must give entName or repoName', ['entName', 'repoName']);
```

Comment:
i dont think you can get enterprise name from the first part of the repo can you?
```suggestion
  const { entName, parentLog = rootLog } = opts;
  if (!entName) // TODO we could get this from installation
    throw new ArgumentError('Must give entName', ['entName']);
```